src/style.css

```diff
@@ -0,0 +1,96 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+*,
+*::after {
+  box-sizing: border-box;
+}
+
+a {
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  list-style-type: none;
+}
+
+.header {
+  margin: 0 auto;
+}
+
+.header__inner {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: auto 50px;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  margin: 0 10px;
+}
+
+.nav__link {
+  position: relative;
+  display: inline-block;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  font-size: 12px;
+  line-height: 14.06px;
+  padding: 23px 0;
```

Comment:
23 its magic number
https://medium.com/javascript-in-plain-english/is-magic-number-always-a-magic-number-c3b340b3f6cd
center your link not add paddings, make it centered so height does not has any effect on it

Reply:
Thanks. I fixed it. But in result 4 tests are failed  :c

Reply:
@vsm2k17 , never mind! I've already found a resolve